main/acle.md

```diff
@@ -9546,25 +9680,2141 @@ possible to write these operations using normal C arithmetic. For example:
+  //   _za32[_u32]
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svadd_write_za32[_s32]_vg1x2(uint32_t slice,
+                                    svint32x2_t zn, svint32x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_s32]
+  //   _za32[_u32]
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svadd_write_za32[_s32]_vg1x4(uint32_t slice,
+                                    svint32x4_t zn, svint32x4_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_s32]
+  //   _za32[_u32]
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svsub_write_za32[_u32]_vg1x2(uint32_t slice,
+                                    svuint32x2_t zn, svuint32x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_s32]
+  //   _za32[_u32]
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svsub_write_za32[_u32]_vg1x4(uint32_t slice,
+                                    svuint32x4_t zn, svuint32x4_t zm);
+  
+  ```
+
+#### ADD (vectors)
+
+Multi-vector add
+
+``` c
+  // Variants are also available for _single_u8_x2, _single_s16_x2,
+  // _single_u16_x2, _single_s32_x2, _single_u32_x2, _single_s64_x2 and
+  // _single_u64_x2
+  __attribute__((arm_streaming))
+  svint8x2_t svadd[_single_s8_x2](svint8x2_t zdn, svint8_t zm);
+  
+  
+  // Variants are also available for _single_u8_x4, _single_s16_x4,
+  // _single_u16_x4, _single_s32_x4, _single_u32_x4, _single_s64_x4 and
+  // _single_u64_x4
+  __attribute__((arm_streaming))
+  svint8x4_t svadd[_single_s8_x4](svint8x4_t zdn, svint8_t zm);
+  
+  ```
+
+#### ADD, SUB, FADD, FSUB (accumulate into ZA)
+
+Multi-vector add/sub and accumulate into ZA
+
+``` c
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za32[_s32]
+  //   _za32[_u32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svadd_za32[_f32]_vg1x2(uint32_t slice, svfloat32x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
+  //   _za32[_s32]
+  //   _za32[_u32]
+  //   _za64[_f64] (only if __ARM_FEATURE_SME_F64F64 != 0)
+  //   _za64[_s64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u64] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svadd_za32[_f32]_vg1x4(uint32_t slice, svfloat32x4_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_f32]
```

Comment:
I think these svsubs should have s32 variants too, since the underlying instruction is sign-agnostic.